frontend/awx/resources/templates/TemplatePage/TemplateLaunchWizard.tsx

```diff
@@ -0,0 +1,319 @@
+import { usePostRequest } from '../../../../common/crud/usePostRequest';
+import {
+  LoadingPage,
+  PageHeader,
+  PageLayout,
+  PageWizard,
+  PageWizardStep,
+  useGetPageUrl,
+  usePageAlertToaster,
+} from '../../../../../framework';
+import { PageFormCredentialSelect } from '../../credentials/components/PageFormCredentialSelect';
+import { PageFormExecutionEnvironmentSelect } from '../../../administration/execution-environments/components/PageFormExecutionEnvironmentSelect';
+import { PageFormInstanceGroupSelect } from '../../../administration/instance-groups/components/PageFormInstanceGroupSelect';
+import { PageFormInventorySelect } from '../../inventories/components/PageFormInventorySelect';
+import { useLabelPayload } from '../hooks/useLabelPayload';
+import { CredentialPasswordsStep, OtherPromptsStep, TemplateLaunchReviewStep } from './steps';
+import { parseStringToTagArray } from '../JobTemplateFormHelpers';
+
+import type { JobTemplate } from '../../../interfaces/JobTemplate';
+import type { LaunchConfiguration } from '../../../interfaces/LaunchConfiguration';
+import type { UnifiedJob } from '../../../interfaces/UnifiedJob';
+import type { Inventory } from '../../../interfaces/Inventory';
+import type { Credential } from '../../../interfaces/Credential';
+import type { ExecutionEnvironment } from '../../../interfaces/ExecutionEnvironment';
+
+export interface TemplateLaunch {
+  inventory: Inventory;
+  credentials: Credential[];
+  credential_passwords: { [key: string]: string };
+  instance_groups: { id: number; name: string }[];
+  execution_environment: ExecutionEnvironment;
+  diff_mode: boolean;
+  extra_vars: string;
+  forks: number;
+  job_slice_count: number;
+  job_tags: { name: string }[];
+  job_type: string;
+  labels: { name: string; id?: number }[];
+  limit: string;
+  scm_branch: string;
+  skip_tags: { name: string }[];
+  timeout: number;
+  verbosity: number;
+}
+
+interface LaunchPayload {
+  credentials: number[];
+  credential_passwords: { [key: string]: string };
+  diff_mode: boolean;
+  execution_environment: number;
+  extra_vars: string;
+  forks: number;
+  instance_groups: number[];
+  inventory: number;
+  job_slice_count: number;
+  job_tags: string;
+  job_type: string;
+  labels: number[];
+  limit: string;
+  scm_branch: string;
+  skip_tags: string;
+  timeout: number;
+  verbosity: number;
+}
+type LaunchPayloadProperty = keyof LaunchPayload;
+
+export function TemplateLaunchWizard() {
+  const { t } = useTranslation();
+
+  const postRequest = usePostRequest<Partial<LaunchPayload>, UnifiedJob>();
+  const createLabelPayload = useLabelPayload();
+
+  const alertToaster = usePageAlertToaster();
+  const getPageUrl = useGetPageUrl();
+  const navigate = useNavigate();
+  const params = useParams<{ id: string }>();
+  const resourceId = params.id?.toString() ?? '';
+  const { data: template } = useGet<JobTemplate>(`/api/v2/job_templates/${resourceId}/`);
```

Comment:
useGet could also deal with error, and display AwxError, but this has not been a standard across the application.